test/configs/debian8_clang_autoconfig_test.sh

```diff
@@ -63,7 +63,8 @@ tar -xf ${TEST_TMPDIR}/debian8-clang-${CONFIG_VERSION}-bazel_${BAZEL_VERSION}-au
 rm -rf ${TEST_TMPDIR}/local_config_cc/tools ${TEST_TMPDIR}/local_config_cc/WORKSPACE
 
 # Unpack the tarball containing published toolchain configs for Bazel 0.10.0 from GitHub.
-tar -xf ${TEST_SRCDIR}/bazel_toolchains_test/file/${COMMIT}.tar.gz -C ${TEST_TMPDIR}
+# The downloaded file of `http_file` rule is hardcoded as `file/downloaded`.
+tar -xf ${TEST_SRCDIR}/bazel_toolchains_test/file/downloaded -C ${TEST_TMPDIR}
```

Comment:
I dont think the right solution is to hardcode the downloaded name here, you could instead change the target that this depends on (https://github.com/bazelbuild/bazel-toolchains/blob/master/test/configs/BUILD#L146) from @bazel_toolchains_test//file to third_party/bazel_toolchains_test:rename_file (and create a BUILD file in third_party/bazel_toolchains_test:rename_file that has a genrule to rename).